packages/kokkos-kernels/src/sparse/KokkosSparse_gauss_seidel_handle.hpp

```diff
@@ -50,292 +50,292 @@
 
 namespace KokkosSparse{
 
-enum GSAlgorithm{GS_DEFAULT, GS_PERMUTED, GS_TEAM};
+  enum GSAlgorithm{GS_DEFAULT, GS_PERMUTED, GS_TEAM};
```

Comment:
I'm not a fan of the name of this enum, but it was already there, so you're stuck with it.  Enums in a public namespace should have a human-readable name, like `GaussSeidelAlgorithm`.